From: Markus Trippelsdorf <markus@trippelsdorf.de>
Subject: Add missing braces to if statement
Date: Mon, 14 Dec 2015 16:44:03 +0100
Origin: http://article.gmane.org/gmane.linux.kernel/2108038

Add missing braces to if statement.

The issue was pointed out by gcc-6's -Wmisleading-indentation.

Acked-by: Ingo Molnar <mingo@kernel.org>
Signed-off-by: Markus Trippelsdorf <markus@trippelsdorf.de>
---
diff --git a/tools/perf/ui/browsers/annotate.c b/tools/perf/ui/browsers/annotate.c
index d4d7cc27252f..718bd46d47fa 100644
--- a/tools/perf/ui/browsers/annotate.c
+++ b/tools/perf/ui/browsers/annotate.c
@@ -755,11 +755,11 @@ static int annotate_browser__run(struct annotate_browser *browser,
 				nd = browser->curr_hot;
 			break;
 		case K_UNTAB:
-			if (nd != NULL)
+			if (nd != NULL) {
 				nd = rb_next(nd);
 				if (nd == NULL)
 					nd = rb_first(&browser->entries);
-			else
+			} else
 				nd = browser->curr_hot;
 			break;
 		case K_F1:
